gap/attr.gi

```diff
@@ -1687,6 +1687,127 @@ function(D)
   return fail;
 end);
 
+InstallMethod(DigraphCore, "for a digraph",
+[IsDigraph],
+function(digraph)
+  local N, lo, topdown, bottomup, hom, lo_var, image,
+  comps, comp, cores, D, in_core, n, m, L, i;
+  digraph := DigraphImmutableCopy(digraph);
+  # copy is necessary so can change vertex labels in function
+  N := DigraphNrVertices(digraph);
+  if IsEmptyDigraph(digraph) then
+    if N >= 1 then
+      return [1];
+    else
+      return [];
+    fi;
+  fi;
+  SetDigraphVertexLabels(digraph, [1 .. N]);
+  digraph := ReducedDigraph(digraph);  # isolated verts are not in core
+  N       := DigraphNrVertices(digraph);
+  if DigraphHasLoops(digraph) then
+    i := First(DigraphVertices(digraph), i -> i in OutNeighbours(digraph)[i]);
+    return [DigraphVertexLabels(digraph)[i]];
```

Comment:
```suggestion
    return [DigraphVertexLabel(digraph, i)];
```
Same kind of philosophy here. It also fits in nicely with what you've done below, with `OutDegreeOfVertex` and `OutNeighboursOfVertex` 🙂